test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/wwwroot/testRunner.js

```diff
@@ -23,6 +23,14 @@ export function assertGraphicExistsInView(viewId, geometryType, count) {
     }
 }
 
+export function assertKmlLayerExists(viewId) {
+    let view = arcGisObjectRefs[viewId];
+    let layers = view.map.layers.items[0].type;
+    if (layers !== 'kml') {
+        throw new Error(`There are no Layers in this view`);
+    }
+}
```

Comment:
This is actually simpler than the code I used to find the widgets, so nice work!